src/markdown-pages/build-apps/customize-open-source-visualization.mdx

```diff
@@ -0,0 +1,1141 @@
+                  />
+                  <VictoryAnimation duration={1000} data={percent}>
+                    {(percent) => (
+                      <VictoryLabel
+                        textAnchor="middle"
+                        verticalAnchor="middle"
+                        x={CHART_WIDTH / 2}
+                        y={CHART_HEIGHT / 2}
+                        text={`${Math.round(percent)}%`}
+                        style={{ ...baseLabelStyles, fontSize: 45 }}
+                      />
+                    )}
+                  </VictoryAnimation>
+                  <VictoryLabel
+                    text={label}
+                    lineHeight={1}
+                    x={CHART_WIDTH / 2}
+                    y={BOUNDS.Y - LABEL_SIZE}
+                    textAnchor="middle"
+                    style={{ ...baseLabelStyles, fontSize: LABEL_SIZE }}
+                  />
+                </svg>
+              );
+            }}
+          </NrqlQuery>
+        )}
+      </AutoSizer>
+    );
+  }
+}
+
+const EmptyState = () => (
+  <Card className="EmptyState">
+    <CardBody className="EmptyState-cardBody">
+      <HeadingText
+        spacingType={[HeadingText.SPACING_TYPE.LARGE]}
+        type={HeadingText.TYPE.HEADING_3}
+      >
+        Please provide a NRQL query & account ID pair
+      </HeadingText>
+      <HeadingText
+        spacingType={[HeadingText.SPACING_TYPE.MEDIUM]}
+        type={HeadingText.TYPE.HEADING_4}
+      >
+        This Visualization supports NRQL queries with a single SELECT clause
+        returning a percentage value (0 to 100 rather than 0 to 1). For example:
+      </HeadingText>
+      <code>
+        {'FROM Transaction SELECT percentage(count(*), WHERE duration < 0.1)'}
+      </code>
+    </CardBody>
+  </Card>
+);
+```
+
+This will revert the bar corners to the default 90-degrees instead of being rounded. While your local server is running, it automatically recognizes changes to _index.js_. So, view your visualization in your browser to see the update:
+
+![Progress bar with straight corners](../../images/build-an-app/progress-bar-chart-no-corner-radius.png)
+
+Perfect! You cloned and updated the open source **Circular progress bar** visualization from the New Relic One catalog. The only thing left to do is publish your version to the catalog so your accounts can subscribe to it.
+
+</Step>
+
+<Step>
+
+Now that you're ready to publish your Nerdpack, stop your local server with `CTRL+C`.
+
+</Step>
+
+</Steps>
+
+## Add a custom visualization to a dashboard
+
+Publish your version of the **Victory charts** Nerdpack to the catalog. Then subscribe to it and use your visualization in a dashboard.
+
+<Callout variant="tip">
+
+Because you used `nr1 clone` to clone the Nerdpack's repository, your local copy already has its own UUID. This is a prerequisite for publishing your version to the New Relic One catalog. If you used `git clone` to copy, you need to update the Nerdpack's UUID manually:
```

Comment:
This could also be a good place to mention updating the `displayName` if we choose not to do it earlier on. 

It's an adjustment is more conceptually important to the publication stage, so I could imagine that point living here